chore: update focus group doc and add page to onboarding (#1213)
medic/cht-docs: content/en/contribute/medic/onboarding/product-team.md

Author: @andrablaj I created this file super fast with the intention to start moving some topics from [All The Things doc](https://docs.google.com/document/d/1LMCwpDTWgWDwwtPjne37T6znGD7cxazZI7CBzU05jUk). As a compliment to the conversations when onboarding new teammates.

---
title: "Product Team"
linkTitle: "Product Team"
weight: 3
description: >
  Introduction to Product Team 
---

The goal of this page is to convey a general “lay of the land” so someone starting can see a lot of what’s out there related to Product Team without having to be surprised each day as new things pop up.

## About Product Team
The Product Team manages the entire software development life-cycle to understand problems, capture requirements, design and build modular software systems, and document everything along the way.
We achieve this by working with CHT partners and health workers to design, build, and maintain the Community Health Toolkit and its open source tools.

## Composition

> The Product team comprises a community team and 4 Focused Working Groups, each focusing on outcomes to serve different people.

Reviewer: ```suggestion
The Product Team comprises a Community Team and 4 Focused Working Groups, each focusing on outcomes to serve different people.
```

Reviewer: For the sake of consistency with the paragraphs above.

Author: Thank you, fixed!

---
title: "Product Team"
linkTitle: "Product Team"
weight: 3
description: >
  Introduction to Product Team 
---

The goal of this page is to convey a general “lay of the land” so someone starting can see a lot of what’s out there related to Product Team without having to be surprised each day as new things pop up.

## About Product Team

> The Product Team as a whole is focused on digital health tools helping to achieve Universal Health Coverage, and care reaching everyone when and where they need it. 

Reviewer: There is a broader definition of the Product Team's scope [here](https://docs.communityhealthtoolkit.org/contribute/medic/).

Author: Not sure if I did it right, can you please check again?

We achieve this by developing and maintaining the CHT in different ways:
- Building tools
- Building community 
- User research and testing
- Collaborations with partners, usually non-profit organization and government's ministries or institutions. 
- Internal cross-functional collaborations with Programs and Research teams. 

## Composition

> The Product team is made up of a community team and 4 focused groups, with each group focusing on outcomes to serve different people.

Reviewer: ```suggestion
The Product team comprises a community team and 4 Focused Working Groups, each focusing on outcomes to serve different people.
```

> ### Focus Groups

Reviewer: ```suggestion
### Focused Working Groups
```

The Product Team operates in a way that is most commonly referred to as Continuous Discovery. That process helps us to focus on the impact of our work and keep teammates well-aligned with the people who use the CHT. Read more about the process [here]({{< ref "contribute/medic/product-development-process" >}}). 

> Each focus group serve to different user types to understand opportunities for improvement, find solutions that will have an impact, and then build, deploy, and measure them. Find out more about the different focus groups [here]({{< ref "contribute/medic/product-development-process/focused-groups" >}}).

Reviewer: ```suggestion
Each Focused Working Group serves different user types to understand opportunities for improvement, find solutions that will have an impact, and then build, deploy, and measure them. Find out more about the different Focused Working Groups [here]({{< ref "contribute/medic/product-development-process/focused-groups" >}}).
```

> ### Community Team

Author: @esthermmoturi We will have new teammates joining Medic. The intention of this page is to introduce topics related to the Product Team as a compliment to the onboarding process. This is just a start, and we will be adding more things. 
Can you please help me fill up this piece? What can we say about the Community Team to new teammates? Some ideas: 
- What is Community team?
- Why do we have a community team?
- What are Round ups? - link to previous round up calls? 
- etc
